src/plugins/identity/identity.js

```diff
@@ -31,16 +32,30 @@ export type IdentitySpec = {|
 |};
 
 /**
- * Create a new node representing an identity.
+ * Internal method for validating a username.
+ *
+ * Returns the username with any leading @ symbol stripped.
+ * Throws an error if the username is invalid.
  */
-export function identityNode(identity: Identity): Node {
+function validateUsername(username: string): Username {
   const re = new RegExp(USERNAME_PATTERN);
-  const match = re.exec(identity.username);
+  const match = re.exec(username);
   if (match == null) {
-    throw new Error(`Invalid username: ${identity.username}`);
+    throw new Error(`Invalid username: ${username}`);
   }
-  const username = match[1];
-  const address = NodeAddress.append(nodePrefix, username);
+  return match[1];
+}
+
+/**
+ * Create a new node representing an identity.
+ */
+export function identityNode(identity: Identity): Node {
```

Comment:
Not sure if this is the standard naming pattern elsewhere, but it would be more intuitive if it was named "createNodeForIdentity" or "createIdentityNode"

Reply:
re: both naming questions: We don’t generally use `get*` prefixes except
where they are informative (usually, `get` suggests that an operation is
expensive or effectful, like reading from disk or network).

Yes, the pattern is standard (see, e.g., `Graph.node`). These functions
just return values. They don’t “retrieve” or “get” anything from
anywhere. Changing `identityNode` to `createNodeForIdentity` just takes
up space and energy without communicating more. These are fine as is.